src/app/FaceFit/page.tsx

```
@@ -69,20 +69,23 @@ const FaceFit = () => {
           {FaceData.map(({ id }) => (
             <SelectedFaceBox
               onPress={() => setSelectFaceFit(id)}
-              selected={id === selectFaceFit}
-            >
-              {id === selectFaceFit &&
+              selected={id === selectFaceFit}>
+              {id === selectFaceFit && (
                 <CheckBox>
                   <Check size={20} color={color.white} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)